### ISSUE TITLE
1.23.0: error: member access into incomplete type 'QScreen'

### ISSUE THREAD

**Issue by yurivict** [OpenModelica/OpenModelica#12549]
### Description

```
/usr/ports/science/openmodelica/work/OpenModelica-1.23.0/OMEdit/OMEditLIB/Modeling/LibraryTreeWidget.cpp:3532:52: error: member access into incomplete type 'QScreen'
```

This module uses QScreen but there is no corresponding include statement.

Version: 1.23.0
clang-18
FreeBSD 14.1

**Comment by perost** [comment]
Are you using the latest master? This should already have been fixed by #12515.

**Comment by yurivict** [comment]
No, I was using 1.23.0.
Thanks.
